pages/api/admin/setup-links/index.ts

```diff
@@ -49,8 +56,16 @@ const handleDELETE = async (req: NextApiRequest, res: NextApiResponse) => {
 
   const { setupID } = req.query as { setupID: string };
 
+  const setupLink = await setupLinkController.getById(setupID);
```

Comment:
This and the sendAudit can be combined into one "non-awaited operation" that does not block the response handling.